src/index.html

```diff
@@ -7,12 +7,74 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section
+      data-qa="card"
+      class="card"
+    >
+      <div class="card__product">
+        <div class="card__product-img card__product-img--position">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K (MNED2UA/A)"
+            class="card__img"
+          />
+        </div>
+
+        <div class="card__product-name">
+          <h2 class="card__name">
```

Comment:
Consider using a less significant heading tag, like h3 or h4, for the product name. It's not likely the second most important piece of content on a page, which is what h2 represents.